roxie/ccd/ccdlistener.cpp

```diff
@@ -1651,10 +1651,10 @@ class RoxieSocketWorker : public RoxieQueryWorker
                     }
                     else
                     {
-                        StringBuffer querySetName;
-                        queryFactory.setown(globalPackageSetManager->getQuery(queryName, &querySetName, NULL, logctx));
                         if (isHTTP)
                             client->setHttpMode(queryName, isRequestArray, httpHelper.queryContentFormat());
+                        StringBuffer querySetName(httpHelper.queryTarget());
```

Comment:
Is this valid when not in http mode? Would be cleaner to set in the if (isHTTP) branch perhaps?